tests/client/core/TestInstallAddon.js

```diff
@@ -96,6 +97,10 @@ describe('withInstallHelpers inner functions', () => {
     mapDispatchToProps = makeMapDispatchToProps({ WrappedComponent, src });
   });
 
+  beforeEach(() => {
+    configStub = sinon.stub(config, 'get').withArgs('server').returns(false);
```

Comment:
ok, cool, it sounds like the mocking is concurrency aware. We may need to double check that sinon's sandbox also works the same way. If the sinon sandbox gets reset after every test then I think it will be fine.